### ISSUE TITLE
When I save a tensorflow session in windows the file(e.g 1.ckpt.data-00000-of-00001) size becomes around 1.9 GB whereas in linux it is around 270 MB. The file generated in windows gives 2% accuracy(supposed to be 82%) when restored in testing.  So the weights are not getting saved properly and many extra info gets stored in Windows I think.  Kindly help to solve the problem in Windows.

### ISSUE THREAD

**Issue by ani31101993** [tensorflow/tensorflow#20344]
Please go to Stack Overflow for help and support:

https://stackoverflow.com/questions/tagged/tensorflow

If you open a GitHub issue, here is our policy:

1. It must be a bug, a feature request, or a significant problem with documentation (for small docs fixes please send a PR instead).
2. The form below must be filled out.
3. It shouldn't be a TensorBoard issue. Those go [here](https://github.com/tensorflow/tensorboard/issues).

**Here's why we have that policy**: TensorFlow developers respond to issues. We want to focus on work that benefits the whole community, e.g., fixing bugs and adding features. Support only helps individuals. GitHub also notifies thousands of people when issues are filed. We want them to see you communicating an interesting problem, rather than being redirected to Stack Overflow.

------------------------

### System information
- **Have I written custom code (as opposed to using a stock example script provided in TensorFlow)**:
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**:
- **TensorFlow installed from (source or binary)**:
- **TensorFlow version (use command below)**:
- **Python version**: 
- **Bazel version (if compiling from source)**:
- **GCC/Compiler version (if compiling from source)**:
- **CUDA/cuDNN version**:
- **GPU model and memory**:
- **Exact command to reproduce**:

You can collect some of this information using our environment capture script:

https://github.com/tensorflow/tensorflow/tree/master/tools/tf_env_collect.sh

You can obtain the TensorFlow version with

python -c "import tensorflow as tf; print(tf.GIT_VERSION, tf.VERSION)"

### Describe the problem
Describe the problem clearly here. Be sure to convey here why it's a bug in TensorFlow or a feature request.

### Source code / logs
Include any logs or source code that would be helpful to diagnose the problem. If including tracebacks, please include the full traceback. Large logs and files should be attached. Try to provide a reproducible test case that is the bare minimum necessary to generate the problem.

**Comment by ani31101993** [comment]
System information
-Not custom code
-windows server 2016 essentials
-python 3.6.5
-jupyter notebook 5.5.0
-tensorflow 1.2.0

**Comment by ani31101993** [comment]
I have used tf.train.Saver().restore(sess, "1.ckpt") to restore and for saving the model I have used tf.train.Saver().save(sess, "./model/"+str(epoch)+".ckpt")

**Comment by limitless4** [comment]
Any update on this?

**Comment by tensorflowbutler** [comment]
Nagging Assignee @cy89: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by cy89** [comment]
@ani31101993 Tensorflow 1.2.0 is pretty old. Can you please see if your program continues with a more recent version?

**Comment by tensorflowbutler** [comment]
It has been 17 days with no activity and the `awaiting response` label was assigned. Is this still an issue?

**Comment by tensorflowbutler** [comment]
It has been 34 days with no activity and the `awaiting response` label was assigned. Is this still an issue?

**Comment by tensorflowbutler** [comment]
We are closing this issue for now due to lack of activity. Please comment if this is still an issue for you. Thanks!
